_posts/2018-07-31-how-to-move-to-https.md

```diff
@@ -0,0 +1,100 @@
+
+### Install the certificate
+
+![Lots of wires]({{ site.url }}/assets/img/blog/install-https-analogy.jpg)
+
+Installing an HTTPS certificate manually on a server is a non-trivial engineering task. We [explain this at a high-level in part one](/2018/06/26/why-you-need-https.html#what-authentication-does-for-you). It  requires expertise from someone who is experienced and comfortable administering servers. There are many different installation methods unique to each permutation of server software and hosting platform, so I won't expend any real estate here attempting to lay it out. If you have to do a manual installation, it's best to search your hosting provider's documentation. However, depending on the complexity of your website architecture, there are ways to ease this process. Some hosting platforms have tools that substantially simplify the installation process. More on _that_ in a moment as well.
+
+### Serve all resources over HTTPS: avoid mixed content
+
+Once you've installed your certificate, it's time to ensure all assets served from your pages are served over HTTPS. Naturally, this entire process should be completed in a staging environment before making the switch to your production environment. Completing a full transition to HTTPS requires attention to detail and diligence. "Mixed content", or serving assets from a page over both HTTP _and_ HTTPS, can be both tedious and insidious to rectify. The longer your site has been around, the more content there is and the more historic hands have been in the pot of content creation, the more work there will be to make the switch. Depending on your platform (CMS or otherwise) and how it was designed, there may be many avenues for different stakeholders to have included assets within a page over time. Developers, site admins, and content editors usually have the ability to add assets to a page. If any assets start with `http://`, they'll need to be updated to `https://` to prevent mixed content warnings.
+
+We have recently helped a client who has been publishing articles at a high cadence for over 10 years with many different stakeholders over that period. Practices weren't consistent and uncovering all the ways in which HTTP resources were served from the page was a substantial undertaking. Therefore, be prepared for a time investment here, there may be many areas to audit to ensure all assets from all your pages are being served over HTTPS. Some common ways to experience mixed content in which HTTP assets are served from a site whose HTML is served over HTTPS are:
+
```

Comment:
"Some common ways mixed content HTTP assets are served from a site whose HTML is served over HTTPS are:"